File: client-app/src/app/models/doktori.ts
import { Pacienti } from "./pacienti";
import { PacientiDoktoriDTO } from "./PacientiDoktoriDTO";
import { Tretmani } from "./tretmani";
import { User, UserFormValues } from "./user";



export interface Doktori extends User {
    kualifikimi: string;
    specializimi: string;
    pacientiDoktori?: PacientiDoktoriDTO[];
    tretmanet?: Tretmani[];

    pacientet?: Pacienti[];
    
  }
  export interface DoktoriFormValues extends UserFormValues{
    kualifikimi?: string;
    specializimi?: string;

    pacientet?: Pacienti [];
  }
  

File: Domain/Doktori.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Domain
{
    public class Doktori : AppUser
    {
      
        public string Kualifikimi { get; set; }
        public string Specializimi { get; set; }
        public ICollection<PacientiDoktori> Pacientet { get; set; } = new List<PacientiDoktori>();
        public ICollection<Tretmani> Tretmanet { get; set; } = new List<Tretmani>();
        public ICollection<Termini> Terminet { get; set; } = new List<Termini>();
    }
}